packages/edit-navigation/src/components/header/manage-locations.js

```diff
@@ -34,7 +34,7 @@ export default function ManageLocations() {
 			labelPosition="top"
 			value={ menuLocation.menu }
 			options={ [
-				{ value: 0, label: __( '-' ) },
+				{ value: 0, label: __( '— Select a Menu —' ) },
```

Comment:
```suggestion
				{ value: 0, label: __( 'Select a Menu' ) },
```

Looking at [the i18n guidelines](https://codex.wordpress.org/I18n_for_WordPress_Developers#Best_Practices) I'm not sure we should include the characters inside the translated string.